roxctl/central/export/export.go

```diff
@@ -0,0 +1,25 @@
+package export
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/roxctl/central/export/deployments"
+	"github.com/stackrox/rox/roxctl/central/export/images"
+	"github.com/stackrox/rox/roxctl/central/export/nodes"
+	"github.com/stackrox/rox/roxctl/central/export/pods"
+	"github.com/stackrox/rox/roxctl/common/environment"
+)
+
+// Command defines the central command tree
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	c := &cobra.Command{
+		Use:   "export",
+		Short: "Commands related to exporting data from Central.",
```

Comment:
The changelog mentions that this is currently tech preview, might be worthwhile to also have this in the help outlined (I suppose https://github.com/stackrox/stackrox/blob/master/roxctl/netpol/netpol.go#L17-L21 is a good example, and would be worthwhile to factor out as a default for other commands to use).